pyuntl/dc_structure.py

```diff
@@ -248,6 +248,8 @@ def identifier_director(**kwargs):
     """Direct how to handle the identifier element."""
     ark = kwargs.get('ark', None)
     domain_name = kwargs.get('domain_name', None)
+    # Set default scheme if it is None or is not supplied.
+    scheme = kwargs.get('scheme') or 'http'
```

Comment:
For consistency with the other changes you've made, can we make this `scheme = kwargs.get('scheme', 'http')`?